quarkus/integration-tests/embedded/src/test/java/org/infinispan/quarkus/embedded/InfinispanEmbeddedTestResource.java

```diff
@@ -15,7 +15,7 @@
 public class InfinispanEmbeddedTestResource implements QuarkusTestResourceLifecycleManager {
     @Override
     public Map<String, String> start() {
-        String xmlLocation = Paths.get(System.getProperty("build.directory"), "classes", "embedded.xml").toString();
+        String xmlLocation = Paths.get("src", "main", "resources", "embedded.xml").toString();
```

Comment:
I still think we should go with a filtered config file, so that it can use the schema headers to validate its compatibility with the latest schema